desc/objectives/_neoclassical.py

```diff
@@ -248,3 +254,253 @@ def compute(self, params, constants=None):
+        Must be broadcastable to Objective.dim_f
+    normalize : bool, optional
+        Whether to compute the error in physical units or non-dimensionalize.
+    normalize_target : bool, optional
+        Whether target and bounds should be normalized before comparing to computed
+        values. If `normalize` is `True` and the target is in physical units,
+        this should also be set to True.
+    loss_function : {None, 'mean', 'min', 'max'}, optional
+        Loss function to apply to the objective values once computed. This loss function
+        is called on the raw compute value, before any shifting, scaling, or
+        normalization.
+    deriv_mode : {"auto", "fwd", "rev"}
+        Specify how to compute Jacobian matrix, either forward mode or reverse mode AD.
+        "auto" selects forward or reverse mode based on the size of the input and output
+        of the objective. Has no effect on self.grad or self.hess which always use
+        reverse mode and forward over reverse mode respectively.
+    rho : ndarray
+        Unique coordinate values specifying flux surfaces to compute on.
+    alpha : ndarray
+        Unique coordinate values specifying field line labels to compute on.
+    num_transit : int
+        Number of toroidal transits to follow field line.
+        For axisymmetric devices, one poloidal transit is sufficient. Otherwise,
+        more transits will give more accurate result, with diminishing returns.
+    knots_per_transit : int
+        Number of points per toroidal transit at which to sample data along field
+        line. Default is 100.
+    num_quad : int
+        Resolution for quadrature of bounce integrals. Default is 32.
+    num_pitch : int
+        Resolution for quadrature over velocity coordinate. Default is 64.
+    batch : bool
+        Whether to vectorize part of the computation. Default is true.
+    num_well : int
+        Maximum number of wells to detect for each pitch and field line.
+        Default is to detect all wells, but due to limitations in JAX this option
+        may consume more memory. Specifying a number that tightly upper bounds
+        the number of wells will increase performance.
+    name : str, optional
+        Name of the objective function.
+    Nemov : bool
+        Whether to use the Γ_c as defined by Nemov et al. or Velasco et al.
+        Default is Nemov. Set to ``False`` to use Velascos's.
+
+        Note that Nemov's Γ_c converges to a finite nonzero value in the
+        infinity limit of the number of toroidal transits.
+        Velasco's expression is defined to be zero on irrational surfaces;
+        and therefore, the numerical computation will converge to zero as the
+        number of toroidal transits increases. This is mentioned to remind
+        users that an optimization using Velasco's metric should be evaluated by
+        measuring decrease in Γ_c at a fixed number of toroidal transits.
```

Comment:
@PlasmaControl/desc-dev because grad(alpha) -> infinity and the bounce averages of radial/poloidal drift will therefore tend to 0. see first few pages of Nemov's gamma c to see how he handles this issue